cmd/blockchyp/main.go

```diff
@@ -565,6 +568,29 @@ func processGiftActivate(client *blockchyp.Client, args blockchyp.CommandLineArg
 	dumpResponse(&args, res)
 }
 
+func processEnroll(client *blockchyp.Client, args blockchyp.CommandLineArguments) {
+	if (args.TerminalName == "") && (args.Token == "") {
+		fatalError("-terminal or -token requred")
+	}
+	req := blockchyp.EnrollRequest{}
+	req.TerminalName = args.TerminalName
+	req.TransactionRef = args.TransactionRef
+	req.Test = args.Test
+	req.ManualEntry = args.ManualEntry
+
+	res, err := client.Enroll(req)
+
+	if err != nil {
+		if res == nil {
```

Comment:
What is this condition checking for? Both decision trees do the exact same thing. This could be represented as:

```
if res == nil || len(res.ResponseDescription) == 0 {
    handleError(&args, err)
}
```
But this still looks like it doesn't do anything meaningful to me. There's a big hole where we don't call handleError at all, and I think we should be calling it no matter what if err != nil?